File: CollegeDatabasePython/Student.py
from AbstractPerson import AbstractPerson
import string

class Student(AbstractPerson):
    
    def __init__(self, firstName, lastName, groupId):
        AbstractPerson.__init__(self, firstName, lastName)
        self.GroupId = groupId

    def GroupId(self):
        return self.GroupId

    def ToString(self):
        return "Student[Id = {0}, First Name = {1}, Last Name {2}, GroupId = {3}]".format(self.Id, self.FirstName, self.LastName, self.GroupId);

File: CollegeDatabase/Persons/Student.cs
using System;

namespace CollegeDatabase.Persons
{
    public class Student : AbstractPerson
    {
        // Dodatkowe pole dla Studenta
        public Guid GroupId { get; set; }

        public Student(string firstName, string lastName, Guid groupId)
        {
            this.Id = Guid.NewGuid();
            this.FirstName = firstName;
            this.LastName = lastName;
            this.GroupId = groupId;
        }

        public override string ToString()
        {
            return $"Student[Id = { this.Id }, First Name = { this.FirstName }, Last Name { this.LastName }, Group Id = { this.GroupId }]";
        }
    }
}